widget-showcase/src/main.ts

```diff
@@ -0,0 +1,9 @@
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import App from './App';
+import dojoTheme from '@dojo/widgets/themes/dojo/theme';
+
+const root = document.querySelector('showcase') || undefined;
```

Comment:
Let's remove the query selector and just do `projector.append()`, which will append to the body. Also means we can remove `<showcase></showcase>` from the index.html

Reply:
Ok done. Personally I find it a bit confusing to see an empty HTML file and it's unclear what `projector.append()` does exactly!